tools/RAiDER/cli/raider.py

```diff
@@ -610,8 +609,7 @@ def calcDelaysGUNW(iargs: Optional[list[str]] = None) -> Optional[xr.Dataset]:
         args.interpolate_time == 'azimuth_time_grid'
     ):
         gunw_id = args.file.name.replace('.nc', '')
-        weather_model_name = identify_which_hrrr(args.file)
-        if not RAiDER.aria.prepFromGUNW.check_hrrr_dataset_availablity_for_s1_azimuth_time_interpolation(gunw_id, weather_model_name):
+        if not RAiDER.aria.prepFromGUNW.check_hrrr_dataset_availablity_for_s1_azimuth_time_interpolation(gunw_id):
```

Comment:
@sssangha this seems like an odd change for this PR. Was this causing an error for you due to a change in the underlying function?

Reply:
Whoops, this was associated with some divergence in my local code from a previous version of the code.

I've updated this to remove such divergence.